wordpress-mobile/WordPress-iOS
# Reader: Enable comments button for Jetpack sites

Fixes #7660 

To test:
1. Follow a Jetpack site with comments enabled
2. Reader->Followed Sites->check that the comment button is there
3. Check that you can post a comment

Needs review: @aerych
